atws/connection.py

```diff
@@ -74,10 +81,15 @@ def get_connection_url(**kwargs):
     except (KeyError):
         url = None
     if not url:
-        url = get_zone_wsdl(kwargs['username'])
+        apiversion = 1.5
```

Comment:
you get an stacktrace if apiversion isn't specified 
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/atws/connection.py", line 86, in get_connection_url
    apiversion = kwargs['apiversion']
KeyError: 'apiversion'
wouldn't it be better to do something like 
if apiversion in kwargs:
 apiversion = kwargs['apiversion']
else:
 apiversion = 1.5
 logging.warning('apiversion not specified, defaulting to 1.5')

Reply:
Right you are.